app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt

```diff
@@ -67,17 +67,17 @@ class SyncIndicatorView(
     )
   }
 
-  private val binding = SyncIndicatorOldBinding.inflate(LayoutInflater.from(context),
-      this,
-      true)
+  private val binding = ViewSyncIndicatorBinding.inflate(LayoutInflater.from(context), this)
 
   private val rootLayout
-    get() = binding.rootLayout
+    get() = binding.root
+
+  private val statusImageView
+    get() = binding.statusImageView
 
   private val statusTextView
     get() = binding.statusTextView
 
-  @SuppressLint("CheckResult")
```

Comment:
Why did we remove this?

Reply:
We don't have any results of Rx stream subscriptions inside `onFinishInflate`, so we don't need it anymore.